src/components/todoFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import { useContext } from "react";
+import cn from "classnames";
+import { TodosContext } from "../services/Store";
+import { Status } from "../types/Status";
+
+export const TodoFooter: React.FC = () => {
+  const { filter, setFilter, todos } = useContext(TodosContext);
+
+  const activeTodos = todos.filter((todo) => !todo.completed).length;
+
+  return (
+    <>
```

Comment:
looks redundant to use fragments here

https://react.dev/reference/react/Fragment#usage